c_emulator/CMakeLists.txt

```
@@ -76,7 +76,7 @@ foreach (xlen IN ITEMS 32 64)
 
             install(TARGETS riscv_sim_${arch}
                 OPTIONAL
-                RUNTIME DESTINATION "bin"
+                RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
```

Actually I think we only need

```suggestion
                RUNTIME
```

It uses `CMAKE_INSTALL_BINDIR` by default.